### PR TITLE
Feat/create inventory part search

### PR DESCRIPTION
**Pull request by ymansfarroll**
Take a look inside :)

### CODE REVIEW

File: package.json
Comment (LuisUlloaC) [inline package.json:83]: It is good practice to always talk to the codeowner before making a change to dependencies of this type

File: prisma/seeders/part.seed.ts
Comment (LuisUlloaC) [inline prisma/seeders/part.seed.ts:1]: You can update this file and improve it using the Faker library. It is already implemented and facilitates the creation of data in large volumes

File: src/lib/features/search-part/helpers/constants.ts
Comment (LuisUlloaC) [inline src/lib/features/search-part/helpers/constants.ts:1]: If it is an empty file then there is no need for its existence

File: package.json
Comment (ymansfarroll) [reply package.json:83]: My apologies for that, it was vital to use the updated features of this module, I'll take ur advice from now on

File: src/routes/(authenticated)/dashboard/maintenance/inventory/+page.server.ts
Comment (LuisUlloaC) [inline src/routes/(authenticated)/dashboard/maintenance/inventory/+page.server.ts:1]: Requests should not be made from the server to the server. The +page.server.ts file is executed on the server side.

**Review by LuisUlloaC — CHANGES_REQUESTED** [review]
Please check the comments and update with the latest from the master branch and verify that there are no errors before requesting another review to do the merge.

File: src/routes/(authenticated)/dashboard/maintenance/inventory/+page.server.ts
Comment (LuisUlloaC) [reply src/routes/(authenticated)/dashboard/maintenance/inventory/+page.server.ts:1]: I have the impression that this is why you had to disable the hooks function

File: src/lib/features/search-part/helpers/constants.ts
Comment (ymansfarroll) [reply src/lib/features/search-part/helpers/constants.ts:1]: It's seems that I forgot to delete it after refactoring the feature

File: prisma/seeders/part.seed.ts
Comment (vladimir1284) [reply prisma/seeders/part.seed.ts:1]: @ymansfarroll no tomes esto en cuenta pues Luis se estña ocupando del seed

File: src/routes/(authenticated)/dashboard/maintenance/inventory/+page.server.ts
Comment (ymansfarroll) [reply src/routes/(authenticated)/dashboard/maintenance/inventory/+page.server.ts:1]: Ya modifique el codigo, revisar y si es necesaria alguna otra modificacion, dejame saber...

**Review by LuisUlloaC — APPROVED** [review]

**Pull request merged**
